test/hotspot/jtreg/compiler/vectorapi/VectorFPtoIntCastTest.java

```diff
@@ -105,7 +105,7 @@ public void checkf2int(int len) {
 
     @Test
     @IR(counts = {IRNode.VECTOR_CAST_F2L, IRNode.VECTOR_SIZE_8, "> 0"},
-        applyIfCPUFeature = {"avx512dq", "true"})
+        applyIfCPUFeatureOr = {"avx512dq", "true", "avx10_2", "true"})
```

Comment:
avx10_2 is super set of AVX512DQ, we enable all AVX512 featurs during VM initialization and IRFrameWork rely on the same.

Reply:
I updated the checks to account for this.